Modules/Common/src/BigScreenConfig.cxx

```diff
@@ -15,83 +15,36 @@
 ///
 
 #include "Common/BigScreenConfig.h"
+#include <CommonUtils/StringUtils.h>
 #include <boost/property_tree/ptree.hpp>
-#include <chrono>
-
-static void splitDataSourceName(std::string s, std::string& det, std::string& name)
-{
-  std::string delimiter = ":";
-  size_t pos = s.find(delimiter);
-  if (pos != std::string::npos) {
-    det = s.substr(0, pos);
-    s.erase(0, pos + delimiter.length());
-  }
-  name = s;
-}
 
 namespace o2::quality_control_modules::common
 {
 
 BigScreenConfig::BigScreenConfig(std::string name, const boost::property_tree::ptree& config)
   : PostProcessingConfig(name, config)
 {
-  auto getParInt = [&](std::string name, int& par) {
-    if (!getConfigParameter(name).empty()) {
-      par = std::stoi(getConfigParameter(name));
-    }
-  };
-
-  // parameters
-  if (const auto& customConfigs = config.get_child_optional("qc.postprocessing." + name + ".customization"); customConfigs.has_value()) {
-    for (const auto& customConfig : customConfigs.value()) {
-      if (const auto& customNames = customConfig.second.get_child_optional("name"); customNames.has_value()) {
-        mConfigParameters.insert(std::make_pair(customConfig.second.get<std::string>("name"), customConfig.second.get<std::string>("value")));
-      }
-    }
-  }
-
-  getParInt("NRows", mNRows);
-  getParInt("NCols", mNCols);
-  getParInt("BorderWidth", mBorderWidth);
-  getParInt("NotOlderThan", mNotOlderThan);
-  getParInt("IgnoreActivity", mIgnoreActivity);
-
-  auto now = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
-
   for (const auto& dataSourceConfig : config.get_child("qc.postprocessing." + name + ".dataSources")) {
     if (const auto& sourceNames = dataSourceConfig.second.get_child_optional("names"); sourceNames.has_value()) {
       for (const auto& sourceName : sourceNames.value()) {
-        std::string sourceDet, sourcePath;
-        splitDataSourceName(sourceName.second.data(), sourceDet, sourcePath);
-        if (sourceDet.empty()) {
+        auto tokens = o2::utils::Str::tokenize(sourceName.second.data(), ':', false, true);
+        if (tokens.size() != 2 || tokens[0].empty()) {
           continue;
         }
-        dataSources.push_back({ sourceDet, sourcePath });
+        dataSources.push_back({ tokens[0], tokens[1] });
```

Comment:
```suggestion
        const auto& sourceDet = tokens[0];
        const auto& sourcePath = tokens[1];
        dataSources.push_back({ sourceDet, sourcePath });
```
one could still keep clear what the tokens stand for

Reply:
Again good point, done as well.